### PR TITLE
dev-python/elasticsearch-py: version bump to 5.0.1

### PR DESCRIPTION
**Pull request by hydrapolic**
- tests now run again
- link to es in chroot bug

### CODE REVIEW

**Comment by gktrk** [comment]
@gentoo/proxy-maint

File: dev-python/elasticsearch-py/elasticsearch-py-5.0.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+# tests fail in chroot
+# https://github.com/elastic/elasticsearch/issues/12018
+RESTRICT="test"
+
+MY_PN=${PN/-py/}
+
+DESCRIPTION="official Python low-level client for Elasticsearch"
+HOMEPAGE="http://elasticsearch-py.rtfd.org/"
+SRC_URI="https://github.com/elasticsearch/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz
+		test? ( https://download.elasticsearch.org/elasticsearch/release/org/elasticsearch/distribution/tar/elasticsearch/${ES_VERSION}/elasticsearch-${ES_VERSION}.tar.gz )"
```
Comment (SoapGentoo) [inline dev-python/elasticsearch-py/elasticsearch-py-5.0.1.ebuild]: remove one tab

File: dev-python/elasticsearch-py/elasticsearch-py-5.0.1.ebuild [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+	# run Elasticsearch instance on custom port
+	sed -i "s/#http.port: 9200/http.port: ${ES_PORT}/g; \
+		s/#cluster.name: my-application/cluster.name: gentoo-es-py-test/g" \
+		${ES}/config/elasticsearch.yml || die
+
+	# start local instance of elasticsearch
+	${ES}/bin/elasticsearch -d -p ${PID} || die
+
+	local i
+	for i in `seq 10`; do
```
Comment (SoapGentoo) [inline dev-python/elasticsearch-py/elasticsearch-py-5.0.1.ebuild]: `for i in {1..10}; do`

File: dev-python/elasticsearch-py/elasticsearch-py-5.0.1.ebuild [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+		grep -q "started" ${ES_LOG} 2> /dev/null
+		if [ $? -eq 0 ]; then
```
Comment (SoapGentoo) [inline dev-python/elasticsearch-py/elasticsearch-py-5.0.1.ebuild]: use two `[[` square brackets `]]` (bash built-in)

File: dev-python/elasticsearch-py/elasticsearch-py-5.0.1.ebuild [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+	ES_LOG="${ES}/logs/elasticsearch.log"
+	PID="${ES}/elasticsearch.pid"
+
+	# run Elasticsearch instance on custom port
+	sed -i "s/#http.port: 9200/http.port: ${ES_PORT}/g; \
+		s/#cluster.name: my-application/cluster.name: gentoo-es-py-test/g" \
+		${ES}/config/elasticsearch.yml || die
+
+	# start local instance of elasticsearch
+	${ES}/bin/elasticsearch -d -p ${PID} || die
```
Comment (SoapGentoo) [inline dev-python/elasticsearch-py/elasticsearch-py-5.0.1.ebuild]: quote the command (what about spaces in `${ES}`?)

**Comment by gktrk** [comment]
Patrick beat you to it again unfortunately. Closing.

commit f3a2e939f24bc4369f26c10019588db46a576c80
Author: Patrick Lauer <patrick@gentoo.org>
Date:   Mon Nov 28 12:10:08 2016 +0100

    dev-python/elasticsearch-py: Bump
    
    Package-Manager: portage-2.3.2

**Comment by hydrapolic** [comment]
@gktrk, @patricklauer did the bump, but forgot to include the improvements ;)

**Comment by hydrapolic** [comment]
Opening new PR: https://github.com/gentoo/gentoo/pull/3152

**Pull request closed without merging**